### PR TITLE
Describe malicious FP detection

### PR DESCRIPTION
**Pull request by velzevur**
Complementary PR to aeternity/aeternity#3197

### CODE REVIEW

File: node/api/channel_ws_api.md [hunk 1]
```diff
@@ -1033,8 +1059,16 @@ Roles:
    to a `closed` state.
  * `"solo_closing"` - reported by both parties, when the on-chain channel state is detected to transition
    to a proper `solo_closing` state - that is, with the latest known state.
- * `"can_slash"` - reported by both parties, when the on-chain channel state is detected to transition to
-   to an improper `solo_closing` state - that is, when there exists a later mutually authenticated state.
+ * `"can_slash"` or `"can_snapshot"`- reported by both parties, when the
+   on-chain channel state is detected to transition to to an improper state -
```
Comment (uwiger) [inline node/api/channel_ws_api.md:1063]:
```suggestion
   on-chain channel state is seen to transition into an improper state -
```

File: node/api/channels_api_usage.md [hunk 2]
```diff
@@ -3156,14 +3156,39 @@ Where `channel_id` has the correct value of the channel's ID.
...
 channel off-chain state on-chain. That would protect them from malicious
 actions from the other party, namely unilaterally closing the channel with an
-older state. This can be prevented by posting a `channel_snapshot_solo_tx`
-transaction on-chain containing the latest co-authenticated off-chain state -
-this guarantees that an older state can not make it on-chain.
+older state. Another use case would be disputing an on-chain
+`channel_force_progress_tx` that is based on an older state. This can be
+prevented by posting a `channel_snapshot_solo_tx` transaction on-chain
+containing the latest co-authenticated off-chain state - this guarantees that
+an older state can not make it on-chain or with the case of forced progress -
```
Comment (uwiger) [inline node/api/channels_api_usage.md:3163]:
```suggestion
an older state can not make it on-chain or, in the case of forced progress,
```

File: node/api/channels_api_usage.md [hunk 3]
```diff
@@ -3156,14 +3156,39 @@ Where `channel_id` has the correct value of the channel's ID.
...
+it will be replaced.
 
 It is worth mentioning that if the latest off-chain state is already present
 on-chain, the snapshot transaction would not provide any new information
 on-chain, so it would fail to be included in the blockchain.
 
+If the channel is not yet closing and a malicious `channel_force_progress_tx`
+transaction is included on-chain - the client gets notified. The malicious
```
Comment (uwiger) [inline node/api/channels_api_usage.md:3171]:
```suggestion
transaction is included on-chain, the client gets notified. The malicious
```

File: node/api/channels_api_usage.md [hunk 4]
```diff
@@ -3156,14 +3156,39 @@ Where `channel_id` has the correct value of the channel's ID.
...
+forced progress transaction would be one being based upon not on the latest
```
Comment (uwiger) [inline node/api/channels_api_usage.md:3172]:
```suggestion
forced progress transaction would be one based upon an off-chain state
older than the latest one.
```

File: node/api/channels_api_usage.md [hunk 5]
```diff
@@ -3156,14 +3156,39 @@ Where `channel_id` has the correct value of the channel's ID.
...
+off-chain state but on an older one. The specifies that a snapshot could
```
Comment (uwiger) [inline node/api/channels_api_usage.md:3173]:
```suggestion
The report declares that a snapshot could dispute the malicious on-chain transaction.
```

File: node/api/channels_api_usage.md [hunk 6]
```diff
@@ -1782,7 +1782,7 @@ that is not based on the latest off-chain state, our FSM informs us about it
     "data": {
       "info": "can_slash",
       "tx": "tx_+NILAfiEuEBCNNHFxu/R+ypbtOCh7BrA+oFrAYYHzhZTR8BmTOgO2yYKd7lXwU7+xlfvD3Mu9dEeVp1T1aVuH8UPk/7wU9UMuECR/+7ZaF0OqeiRuRVkjsd2aEynOBmBk+tFETlA5H/jNIVB2A1RbKDe8yHGBWpUbWZzLnPXK+pl1Wkml094WxUAuEj4RjkCoQYfhMxkEg4U4IPqO0HIxeove0RQjLQ6xewjm9BZLWhckwKgKhvZeiagtdVqx9aMKxKhw8+hK5cMrAWcpuiI9OVBfAcDKgN0",
-      "type": "channel_offchain_tx"
+      "type": "channel_close_solo_tx"
```
Comment (AleksandarFilipov) [inline node/api/channels_api_usage.md:1785]: is this a seperate fix?

Comment (velzevur) [reply node/api/channels_api_usage.md:1785]: this was a bug in the docs :)

**Pull request merged**